tests/test_20_inventory.py

```diff
@@ -53,8 +53,7 @@ def test_edit_instance(registered_agent, browser, ui_url):
     comment = browser.select("#inputCommentUpdate").get_attribute("value")
     assert "Registered by tests." == comment
     multiselect = MultiSelect(browser, "selectGroupsUpdate")
-    multiselect.toggle().unselect("default")
-    multiselect.toggle().select("default")
+    assert "default" in multiselect.selected[0].text
```

Comment:
Can you try?
```suggestion
    assert multiselect.selected[0].text == "default"
```

Reply:
yes sorry, I made it in pdb but keep "in" in my code.. I make the change